src/style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: rgba(6, 11, 53, 1);
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --color: rgba(97, 96, 112, 1);
+  --button-color: rgba(0, 172, 220, 1);
+  --hover-button-color: rgba(255, 255, 255, 1);
+}
+
+.card {
+  padding: 32px 0 16px;
```

Comment:
card should have paddings for all sides (so you don't need to set margin-right/left for children)